include/dof_utilities.h

```diff
@@ -71,6 +71,37 @@ namespace DOFUtilities
   }
 
   /**
+   *  Extract independent local dofs values
+   *  in a cell and store them in
+   *  std::vector <Tensor <1, n_components, Number> > us
+   *  whose size is number of quadrature points in the cell.
+   *
+   */
+  template <int dim, int spacedim, int n_components, typename Number>
+  void
+  get_values (const FEValuesBase<dim, spacedim> &fe_values,
+              const std::vector<Number> &independent_local_dof_values,
+              std::vector <Tensor <1, n_components, Number> > &us)
```

Comment:
This should really be a `std::vector<number>`, since Tensor<1, n_components> can only be instantiated for n_components <=3...